docs/source/pipelines.rst

```diff
@@ -118,39 +119,43 @@ Key concepts
 
 Usage
 -----
+Model development workflow
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The general workflow for using MLflow Pipelines is as follows:
+The general model development workflow for MLflow Pipelines can be summarized through the following
+4 steps:
```

Comment:
I think the prior text worked fine, albeit with "model development":
`The general model development workflow for using MLflow Pipelines is as follows:`